bathlarp/frontend
# feat(styles): sets up root styles and linked stories

**Issues affected**

Closes #5 

**Description**

* Finishes configuring the base for Tailwind
* Adds the starting points for light, dark and contrast themes
* Adds Storybook stories for colour and typography

**How to test**

Run up Storybook and check that the stories are present/the themes are switchable.

**Screenshots**

![image](https://github.com/bathlarp/frontend/assets/7863997/1085f4ed-b7b7-4de4-90c4-500fe26c5b3f)
![image](https://github.com/bathlarp/frontend/assets/7863997/b1bb1a3c-ef08-4ec1-92d3-e3decdb73f96)

**Any other information?**

I've set this up using css variables rather than Tailwind's built in dark theme handling to make it easier to use multiple themes later. I've also removed Sass but implemented PostCSS nesting which is exactly the same syntax...

**Checklist**

- [x] Have all new components been added to Storybook?
- [ ] Have unit/component/integration tests been written?
- [x] Has documentation been updated?
